Assets/Demo/SimpleController_UsingActions_InAsset.cs

```diff
@@ -8,16 +8,32 @@ public class SimpleController_UsingActions_InAsset : MonoBehaviour
     public float moveSpeed;
     public float rotateSpeed;
     public float burstSpeed;
+    public float jumpForce = 2.0f;
     public GameObject projectile;
 
     public DemoControls controls;
 
     private Vector2 m_Move;
     private Vector2 m_Look;
+    private Vector3 m_Jump;
```

Comment:
No need to have this one.

Reply:
29 is just an assignment and 70 the only read it seems. Doesn't seem like it needs a var outside of the jump response code.